include/graphics.h

```diff
@@ -8550,6 +8550,8 @@ extern const u32 gItemIcon_RoomService[];
 extern const u32 gItemIconPalette_RoomService[];
 extern const u32 gItemIcon_UtilityUmbrella[];
 extern const u32 gItemIconPalette_UtilityUmbrella[];
+extern const u32 gItemIcon_ClearAmulet[];
+extern const u32 gItemIconPalette_ClearAmulet[];
```

Comment:
Please move this to be next to the other Gen 9 items.

Reply:
done

Reply:
my bad for not noticing that :'D

Reply:
Don't worry, I also noticed just now that the order differs between them, but that can be cleaned up later xD